### PR TITLE
zh: TiCDC TLS document

### PR DESCRIPTION
**Pull request by weekface**
### What is changed, added or deleted? (Required)

**Don't merge before** https://github.com/pingcap/tidb-operator/issues/3010

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/enable-tls-between-components.md [hunk 1]
```diff
@@ -807,6 +807,67 @@ summary: 在 Kubernetes 上如何为 TiDB 集群组件间开启 TLS。
 
         创建这个对象以后，`cert-manager` 会生成一个名字为 `${cluster_name}-drainer-cluster-secret` 的 Secret 对象供 TiDB 集群的 Drainer 组件使用。
 
+    - TiCDC 组件的 Server 端证书。
```
Comment (DanielZhangQD) [inline zh/enable-tls-between-components.md:810]: 还有 cfssl 部分需要修改？

File: zh/enable-tls-between-components.md [hunk 2]
```diff
@@ -807,6 +848,67 @@ summary: 在 Kubernetes 上如何为 TiDB 集群组件间开启 TLS。
...
+          issuerRef:
+            name: ${cluster_name}-tidb-issuer
+            kind: Issuer
+            group: cert-manager.io
+        ```
+
+        其中 `${cluster_name}` 为集群的名字：
+
+        - `spec.secretName` 请设置为 `${cluster_name}-ticdc-cluster-secret`；
+        - `usages` 请添加上  `server auth` 和 `client auth`；
```
Comment (ran-huang) [inline zh/enable-tls-between-components.md:893]:
```suggestion
        - `usages` 请添加上 `server auth` 和 `client auth`；
```

File: zh/enable-tls-between-components.md [hunk 3]
```diff
@@ -807,6 +848,67 @@ summary: 在 Kubernetes 上如何为 TiDB 集群组件间开启 TLS。
...
+          - `${cluster_name}-ticdc`
+          - `${cluster_name}-ticdc.${namespace}`
+          - `${cluster_name}-ticdc.${namespace}.svc`
+          - `${cluster_name}-ticdc-peer`
+          - `${cluster_name}-ticdc-peer.${namespace}`
+          - `${cluster_name}-ticdc-peer.${namespace}.svc`
+          - `*.${cluster_name}-ticdc-peer`
+          - `*.${cluster_name}-ticdc-peer.${namespace}`
+          - `*.${cluster_name}-ticdc-peer.${namespace}.svc`
+        - `ipAddresses` 需要填写这两个 IP ，根据需要可以填写其他 IP：
```
Comment (ran-huang) [inline zh/enable-tls-between-components.md:904]:
```suggestion
        - `ipAddresses` 需要填写这两个 IP，根据需要可以填写其他 IP：
```

File: zh/deploy-ticdc.md [hunk 4]
```diff
@@ -73,4 +73,14 @@ summary: 了解如何在 Kubernetes 上部署 TiCDC。
     ]
     ```
 
+    TiCDC 从 v4.0.3 版本开始支持 TLS，TiDB Operator v1.1.3 版本同步支持 TiCDC 开启 TLS 功能。
+
+    如果在创建 TiDB 集群时候开启了 TLS，使用 cdc cli 请携带 TLS 证书相关参数：
```
Comment (ran-huang) [inline zh/deploy-ticdc.md:78]:
```suggestion
    如果在创建 TiDB 集群时开启了 TLS，使用 `cdc cli` 请携带 TLS 证书相关参数：
```

**Pull request merged**